app/src/main/java/io/github/keibai/models/Auction.java

```diff
@@ -73,4 +73,12 @@ public String toString() {
                 ", id=" + id +
                 '}';
     }
+
+    public String getWinnerMessage() {
+        return "You have won in Auction " + this.id;
```

Comment:
Wouldn't this fit best in the activity? A model should avoid returning humanized data such as sentences.

Reply:
I think it doesn't matter, besides it shows me options where I can create the method, and one of the option is in Auction.

Reply:
besides I need the Auction id, which is easier to obtain if I am in the Auction.java